mkdocs.yml

```diff
@@ -95,6 +95,8 @@ nav:
       - Leadership: workflow/leadership.md
 markdown_extensions:
   - admonition
+  - attr_list
+  - md_in_html
```

Comment:
Are these use anywhere?

Reply:
yep, had to add this for the annotation for the Kanban Workflow